go/runtime/scheduling/init.go

```diff
@@ -6,13 +6,14 @@ import (
 	registry "github.com/oasisprotocol/oasis-core/go/registry/api"
 	"github.com/oasisprotocol/oasis-core/go/runtime/scheduling/api"
 	"github.com/oasisprotocol/oasis-core/go/runtime/scheduling/simple"
+	ordered "github.com/oasisprotocol/oasis-core/go/runtime/scheduling/simple/txpool/ordered_map"
```

Comment:
```suggestion
	"github.com/oasisprotocol/oasis-core/go/runtime/scheduling/simple/txpool/orderedmap"
```
(e.g., rename the package to `orderedmap`, also elsewhere)